### DISCUSSION TITLE
A Sentinel-2 Dataset for Uganda

### DISCUSSION THREAD

**Discussion by JonasardoLundUniv** [davidfrantz/force#97] [Show and tell]
A Sentinel-2 Dataset for Uganda
created with FORCE, see https://www.mdpi.com/2306-5729/6/4/35

![image](https://user-images.githubusercontent.com/81447806/119166383-d64d1480-ba5e-11eb-8a1c-173c80a0bd76.png)

Regards
/Jonas Ardö

**Comment by davidfrantz** [comment]
Very cool! Congratulations on the publication!
I have already added it to https://force-eo.readthedocs.io/en/latest/refs-applied.html
Cheers,
David
